CI/mxe/before_install.sh

```diff
@@ -1,12 +1,21 @@
 #!/bin/sh
 
+# steps to upgrade MXE dependencies:
+# 1) Use Debian/Ubuntu system or install one (virtual machines will work too)
+# 2) update following script to include any new dependencies 
+# You can also run it to upgrade existing ones, but don't expect much
+# MXE repository only provides ancient versions for the sake of "stability"
+# https://github.com/vcmi/vcmi-deps-mxe/blob/master/mirror-mxe.sh
+# 3) make release in vcmi-deps-mxe repository using resulting tar archive
+# 4) update paths to tar archive in this script
+
 # Install nsis for installer creation
 sudo add-apt-repository 'deb http://security.ubuntu.com/ubuntu bionic-security main'
 sudo apt-get install -qq nsis ninja-build libssl1.0.0
 
 # MXE repository was too slow for Travis far too often
-wget -nv https://github.com/vcmi/vcmi-deps-mxe/releases/download/2021-02-20/mxe-i686-w64-mingw32.shared-2021-01-22.tar
-tar -xvf mxe-i686-w64-mingw32.shared-2021-01-22.tar
+wget -nv https://github.com/vcmi/vcmi-deps-mxe/releases/download/2022-12-26/mxe-i686-w64-mingw32.shared-2022-12-26.tar
+tar -xvf mxe-i686-w64-mingw32.shared-2022-12-26.tar
```

Comment:
I think MXE upgrade should be a separate PR

Reply:
Maybe. But:
1) Changes in MXE packages are really minor, due to their "as stable as fossils" approach. Only several packages got upgrade to newer "patch" version. I think CMake is the only exception, which went from 3.10 to 3.19
2) Since we use MXE as primary Windows build, this PR is largely pointless if MXE and as result - Windows has translations disabled
3) Since we use MXE prebuilt packages repository, I didn't have an option of only adding new package(s) without upgrading everything else.

Reply:
Toolchain upgrade itself has no relation to translations, that's why I view it as a completely separate change. After that your can simply rebase your translation branches on top of that.
Point 3 I didn't understand tbh.

Reply:
Regarding 3:
Normally, I would have simply added new dependency to the list.
This is what I did with Ubuntu-based CI's
With our MXE setup this is impossible - if you want to add even a single package, you have to upgrade entire dependencies package.

So yeah, I'd also prefer to only add linguist tools to MXE without full upgrade (especially considering how useless it is), but we don't have such option. It's all or nothing.

Reply:
clear, thanks. That's totally fine.